src/footer/footer.html

```diff
@@ -29,33 +29,33 @@
       <div class="section launch">
         <div class="title">Prime Launch</div>
         <div class="body">
-          <a href="#" target="_blank" rel="noopener noreferrer"
-            >Why Prime Launch?</a
-          >
-          <a href="#" target="_blank" rel="noopener noreferrer"
-            >Upcoming Launches</a
-          >
-          <a href="#" target="_blank" rel="noopener noreferrer"
-            >Host a Launch</a
-          >
-          <a href="#" target="_blank" rel="noopener noreferrer"
-            >Prime Launch Docs</a
-          >
+          <a href="#" target="_blank" rel="noopener noreferrer">
```

Comment:
really these anchors should be on one line.  that is how they were.  as you have it, one is introducing whitespace, with potential consequences (that I don't know how to fully predict).  further I think, within reason, one-liners are cleaner in appearance.  Same for all these similar changes across this file.

Reply:
Prettier. I'll increase print width